dcos/dcos-ui
# DCOS-55895: Support editing Quota limits for an existing top-level Group

Closes https://jira.mesosphere.com/browse/DCOS-55895

## Testing
0. In `Config.dev:
```
  uiConfigurationFixture: {
    uiConfiguration: {
      features: {     
        quota: true
...
```
1. Go to services page.
2. Create a group if there are none.
3. Verify that an edit action appears in the action dropdown for top-level groups.
4. Verify that an edit action appears in the top right dropdown after clicking on a top-level-group.
5. Verify that the edit modal works (you can go to the quota tab to confirm if the limits actually changed).
6. Verify that the edit action appears only for top-level groups.
7. Verify that the added tests make sense.

## Trade-offs
None.

## Dependencies
None.

## Screenshots
![Screenshot from 2019-08-05 14-44-36](https://user-images.githubusercontent.com/40791275/62462251-9769a400-b78f-11e9-9b75-a6bf243786a3.png)
